UI changes

hoxhaj opened the pull request:
### Description
<!-- Describe your changes in detail -->
Changed some UI elements and added the dropdown languages selection

### Motivation and Context
<!-- Why is this change required? What problem does it solve? -->
<!-- If it fixes an open issue, please link to the issue here. -->

### How Has This Been Tested?
<!-- Please describe in detail how you tested your changes. -->
<!-- Include details of your testing environment, tests ran to see how -->
<!-- your change affects other areas of the code, etc. -->

### Screenshots (if appropriate):

### Checklist:
<!-- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!-- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [ ] GitHub issue linked
- [ ] The necessary translations have been added for all languages
- [ ] Any documentation has been updated accordingly
- [x] Responsive sizes (Mobile) tested
- [x] Cross Browser tested if necessary
- [ ] Conflicts resolved

edswf1 commented:
thank you! i don't know if this is only an issue on my side, but it seems like a lot of lines where changed only due to indentation.

Which Elements where changed besides language dropdown? (maybe i'm just dumb 😄)

hoxhaj commented:
you are totally right! my ide messed it up and i noticed that only now ;(
anyway changes were made on:

- style.css (added lines 423-436)
- script.js (added lines 230-231, changed lines 238-239)
- guide.html (changed lines 62-70, 383-390)
- index.html (added lines 49-53, removed lines 67, changed lines 98-169, 184, 186, 191, 198, 203, 208, 221, 225, 290-297 )

If that's a problem i will send a new pull request.

patricklodder commented:
> If that's a problem i will send a new pull request.

Would be great if you could do that! Also needs a rebase against current gh-pages branch.
